cmd/homi/setup/flags.go

```diff
@@ -41,24 +41,24 @@ var (
 		Aliases: []string{},
 	}
 
-	cypressTestFlag = &cli.BoolFlag{
-		Name:    "cypress-test",
-		Usage:   "Generate genesis.json similar to the one used for Cypress with shorter intervals for testing",
+	mainnetTestFlag = &cli.BoolFlag{
+		Name:    "mainnet-test",
+		Usage:   "Generate genesis.json similar to the one used for Mainnet with shorter intervals for testing",
 		Aliases: []string{},
 	}
-	cypressFlag = &cli.BoolFlag{
-		Name:    "cypress",
-		Usage:   "Generate genesis.json similar to the one used for Cypress",
+	mainnetFlag = &cli.BoolFlag{
+		Name:    "mainnet",
+		Usage:   "Generate genesis.json similar to the one used for Mainnet",
 		Aliases: []string{},
 	}
-	baobabTestFlag = &cli.BoolFlag{
-		Name:    "baobab-test",
-		Usage:   "Generate genesis.json similar to the one used for Baobab with shorter intervals for testing",
+	testnetTestFlag = &cli.BoolFlag{
+		Name:    "testnet-test",
```

Comment:
Can we still accept "baobab" here? Maybe some have a script using `baobab-test`.

Reply:
fixed a9576d2a